### PR TITLE
Update s3.md

### PR DESCRIPTION
**Pull request by transdigiware**
Added dot_match example

### CODE REVIEW

**Comment by plaindocs** [comment]
Happy to merge this is if you can just resolve the merge conflict first. Thanks.

File: user/deployment/s3.md [hunk 1]
```diff
@@ -268,7 +268,20 @@ deploy:
...
 
-### Using S3-compatible Object Storage
+## Set dot_match flag to upload files starting with a period
+
+S3 uploads can be set to upload all files starting with a `.`
+
+```yaml
+deploy:
+  provider: s3
+  ..
```
Comment (BanzaiMan) [inline user/deployment/s3.md]: Is this meant to be ellipses? I don't think this is necessary.

File: user/deployment/s3.md [hunk 2]
```diff
@@ -268,7 +268,20 @@ deploy:
 ```
 {: data-file=".travis.yml"}
 
-### Using S3-compatible Object Storage
+## Set dot_match flag to upload files starting with a period
+
+S3 uploads can be set to upload all files starting with a `.`
```
Comment (BanzaiMan) [inline user/deployment/s3.md]: I think "all files starting with a `.`" is somewhat misleading. Namely, it reads as though those are the only files that will be uploaded.

May I suggest changing this to: "all files including those starting with a `.`"?

File: user/deployment/s3.md [hunk 3]
```diff
@@ -279,4 +292,4 @@ deploy:
   ..
   endpoint: https://nyc3.digitaloceanspaces.com
 ```
-{: data-file=".travis.yml"}
+{: data-file=".travis.yml"}
```
Comment (BanzaiMan) [inline user/deployment/s3.md]: Not sure how this line is showing up in a diff (I can't detect the white space change), but I do prefer this not being a part of the PR.

File: user/deployment/s3.md (hunk 2 above)
Comment (DrTorte) [reply user/deployment/s3.md]: Agreed, let's do that.

File: user/deployment/s3.md (hunk 1 above)
Comment (DrTorte) [reply user/deployment/s3.md]: Let's take them out. :)

File: user/deployment/s3.md (hunk 3 above)
Comment (DrTorte) [reply user/deployment/s3.md]: Seems to be an oddity from fixing the conflict, but agreed.

File: user/deployment/s3.md (hunk 1 above)
Comment (DrTorte) [reply user/deployment/s3.md]: Looking over the doc I see them used a few times, which I believe was the reason for their original inclusion. I think taking them all out would be beneficial and more consistent with other docs, unless I'm mistaken and there are other places we use them?

**Pull request merged**